pybossa/view/projects.py

```diff
@@ -449,15 +448,17 @@ def clone_project(project, form):
 
     if not is_admin_or_subadmin_and_owner:
```

Comment:
Looks like clone_project can only be called by admins or subadmins. if so, can this check be simplified to
> if current_user.id not in project.owners_ids:
>    proj_dict['info'].pop('ext_config', None)

Reply:
Yes  will do